odoo/modules/module.py

```diff
@@ -311,74 +351,85 @@ def get_module_root(path):
         path = new_path
     return path
 
-def load_information_from_description_file(module, mod_path=None):
+@functools.lru_cache(maxsize=None)
+@tools.func.mockable
+def load_manifest(module, mod_path=None):
```

Comment:
The `mockable` decorator requires you to go through the `lru_cache` wrapper, and spread all your tests with calls like `load_manifest.cache_clear()` that are absolutely critical for the test to be correct.  BTW, the tests are the only places where `cache_clear()` is called.  What about this instead?
```suggestion
@tools.func.mockable
@functools.lru_cache(maxsize=None)
def load_manifest(module, mod_path=None):
```
From the point of view of tests, it is a mockable function, period.
The fact that it has a cache is an implementation detail.